content/blog/2022-08-29-announcing-poetry-1-2-0.md

```diff
@@ -0,0 +1,752 @@
+
+- Keeping it was blocking the development of some features, while slowing down the development of others
+- Projects have long had the time to migrate to Python 3
+
+{{% note %}}
+If you rely on Poetry on a Python 2.7 project, you can still use Poetry 1.1.x, though no more bugfixes will be added to
+1.1 branch.
+{{% /note %}}
+
+### Dropping support for Python 2.7, 3.5 and 3.6 for Poetry itself
+
+Poetry 1.2 drops support for Python 2.7, 3.5 and 3.6, meaning that installing Poetry on those Python versions is now
+unsupported.
+
+{{% note %}}
+This change is about installing and running Poetry itself. Managing projects using Python 3.5 and 3.6, as well as older
+Python 3 versions, is still supported.
+{{% /note %}}
+
+### Dependency groups
+
+Poetry provides a way to **organize** your dependencies by **groups**. For instance, you might have dependencies that
+are only needed to test your project or to build the documentation.
+
+To declare a new dependency group, use a `tool.poetry.group.<group>` section where `<group>` is the name of your
+dependency group (for instance, `test`):
+
+```toml
+[tool.poetry.group.test]  # This part can be left out
+
+[tool.poetry.group.test.dependencies]
+pytest = "^7.1.0"
+pytest-mock = "*"
+```
+
+{{% note %}}
+All dependencies **must be compatible with each other** across groups since they will be resolved regardless of whether
+they are required for installation or not
+(see [Installing group dependencies]({{< relref "#installing-group-dependencies" >}})).
+
+Think of dependency groups as **labels** associated with your dependencies: they don't have any bearings on whether
+their dependencies will be resolved and installed **by default**, they are simply a way to organize the dependencies
+logically.
+{{% /note %}}
+
+The dependencies declared in `tool.poetry.dependencies` are part of an implicit `main` group.
+
+```toml
+[tool.poetry.dependencies]  # Implicit `main` dependency group
+httpx = "*"
+pendulum = "*"
+
+[tool.poetry.group.test.dependencies]
+pytest = "^7.1.0"
+pytest-mock = "*"
+```
+
+Dependency groups, other than the implicit `main` group, must only contain dependencies you need in your development
+process. Installing them is only possible by using Poetry.
+
+To declare a set of dependencies, which add additional functionality to the project during runtime,
+use [extras]({{< relref "docs/1.2/pyproject#extras" >}}) instead. Extras can be installed by the end user using `pip`.
+
+{{% note %}}
+**A note about the `dev-dependencies` section**
+
+Any dependency declared in the `dev-dependencies` section will automatically be added to a `dev` group.
+So the two following notations are equivalent:
+
+```toml
+[tool.poetry.dev-dependencies]
+pytest = "^7.1.0"
+pytest-mock = "*"
+```
+
+```toml
+[tool.poetry.group.dev.dependencies]
+pytest = "^7.1.0"
+pytest-mock = "*"
+```
+
+Poetry will slowly transition away from the `dev-dependencies` notation which will soon be deprecated, so it's advised
```

Comment:
We can already call it "deprecated", can't we?

Reply:
Good call. Added https://github.com/python-poetry/website/pull/68/commits/2a6d0c0c777140e00919ac7f5efff9456a0be11e to update that.